pkg/network/protocols/core/protocol.go

```diff
@@ -172,7 +178,7 @@ func (p *Protocol) onBlockRequest(idBytes []byte, id peer.ID) {
 }
 
 func (p *Protocol) onSlotCommitment(commitmentBytes []byte, id peer.ID) {
-	receivedCommitment, err := model.CommitmentFromBytes(commitmentBytes, p.apiProvider, serix.WithValidation())
+	receivedCommitment, _, err := model.CommitmentFromBytes(commitmentBytes, p.apiProvider, serix.WithValidation())
```

Comment:
we have a `lo.DropCount`

Reply:
As written before, drop the serix option and make it always use `serix.WithValidation()` and make the `CommitmentFromBytes` a curried func that takes the api provider so it looks like:
`model.CommitmentFromBytes(p.apiProvider)(commitmentBytes)`